src/accelerate/utils/imports.py

```diff
@@ -63,13 +63,10 @@ def is_torch_distributed_available() -> bool:
 
 
 def is_xccl_available():
-    # Currently IPEX uses custom "ccl" distributed backend. Return False for "xccl"
-    # here to avoid collisions.
+    if is_torch_version(">=", "2.7.0"):
+        return torch.distributed.distributed_c10d.is_xccl_available()
     if is_ipex_available():
         return False
-    # TODO: switch to is_torch_version() once torch 2.7 will be released
-    if version.parse(torch.__version__).release >= version.parse("2.7").release:
-        return torch.distributed.distributed_c10d.is_xccl_available()
```

Comment:
@SunMarc 2.7 rls-ed, so complete this TODO. And since `xpu` already built-in device of `pytorch`, so put `ipex` to second rather than first priority to check. We are planning to retire `ipex` from `transformers` and `accelerate` since pytorch xpu is more and more mature, line of sight is `2.9`.